ocs_sample_library_preview/Security/Owner.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from .TrusteeType import TrusteeType
+
+class Owner(object):
+  """OCS owner definition"""
+
+  def __init__(self, type: TrusteeType = None, tenant_id: str = None, object_id: str = None):
+      self.Type = type
+      self.TenantId = tenant_id
+      self.ObjectId = object_id
+
+  @property
+  def Type(self) -> TrusteeType:
+      return self.__type
+
+  @Type.setter
+  def Type(self, value: TrusteeType):
+      self.__type = value
+
+  @property
+  def TenantId(self) -> str:
+      return self.__tenant_id
+
+  @TenantId.setter
+  def TenantId(self, value: str):
+      self.__tenant_id = value
+
+  @property
+  def ObjectId(self) -> str:
+      return self.__object_id
+
+  @ObjectId.setter
+  def ObjectId(self, value: str):
+      self.__object_id = value
+
+  def toJson(self):
+      return json.dumps(self.toDictionary())
+
+  def toDictionary(self):
+      result = {'Type': self.Type.value}
+
+      if self.TenantId is not None:
+          result['TenantId'] = self.TenantId
+
+      if self.ObjectId is not None:
+          result['ObjectId'] = self.ObjectId
+
+      return result
+
+  @staticmethod
+  def fromJson(content: dict[str, str]):
+      result = Owner()
+
+      if not content:
+          return result
+
+      if 'Type' in content:
+          if (type(content['Type']) is str):
+            result.Type = TrusteeType[content['Type']]
+          else:
+            result.Type = TrusteeType(content['Type'])
```

Comment:
if some endpoints are returning ints and some strings, has it been mentioned to the teams? I'm not sure if we've been doing that for the other times that int <--> string indexing has changed or been different.

Reply:
I asked the Assets team about this. Apparently, architecture made the decision that all enums should be in their string representation when returned by the API. Assets is newer, which is why it is a string enum and older routes such as Streams are planning to switch to strings in the future.